parts/kubernetesagentvars.t

```diff
@@ -7,9 +7,9 @@
     "{{.Name}}AvailabilitySet": "[concat('{{.Name}}-availabilitySet-', variables('nameSuffix'))]",
 {{if .IsWindows}}
     "winResourceNamePrefix" : "[substring(variables('nameSuffix'), 0, 5)]",
-    "{{.Name}}VMNamePrefix": "[concat(variables('winResourceNamePrefix'), variables('orchestratorName'), add(900,variables('{{.Name}}Index')))]",
+    "{{.Name}}VMNamePrefix": "[toLower(concat(variables('winResourceNamePrefix'), variables('orchestratorName'), add(900,variables('{{.Name}}Index'))))]",
```

Comment:
why?

Reply:
this ensures we never hit this problem again with Kubernetes.

Reply:
@JackQuincy the Azure cloud provider in Kubernetes converts node/VM names to lowercase. This led to a bug with some of our ACS infrastructure. This change ensures VM names are consistent at all places (Azure, Kubernetes, infra).

Reply:
@amanohar @jackfrancis thoughts is this over validation. I feel like this is asking for things to get out of sync in code and us to need to use equalsfold all over the place.